scripts/MAC Season 5.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "MAC Season 5 Metadata Upload to bety"
+output: html_document
+---
+
+
+########### Connect to bety database
+```{r dbcon-1, message = FALSE}
+library(RPostgreSQL)
+dbcon <- dbConnect(RPostgreSQL::PostgreSQL(),
+                   dbname = "bety", 
+                   password = 'bety', 
+                   host = 'localhost', 
+                   user = 'bety', 
+                   port = 5432)
+```
+
+
+########### Insert new experiment to experiments table
+```{sql connection = dbcon}
+insert into experiments (name, start_date, end_date, description, design, user_id) 
+values ('MAC Season 5: Durum Wheat', '2017-12-01', '2018-03-31', '', '', 6000000004)
+```
+
+########### Associate new experiment id with site ids
+```{sql connection = dbcon}
+insert into experiments_sites (experiment_id, site_id) 
+   select e.experiment_id, s.site_id 
+        from (select id as experiment_id from experiments where name = 'MAC Season 5: Durum Wheat')  as e 
+     cross join 
+         (select id as site_id from sites where sitename like 'MAC Field Scanner Season 5%') as s
+```
+
+########### Check to see if season 5 sitenames have already been uploaded
+```{r s5_site_query, message = FALSE}
+#get data frame containing all sitenames already present
+sitename_query <- dbGetQuery(dbcon, 'select sitename from sites')
+#convert to vector
+query_sites <- sitename_query$sitename
+
+#read in the Season 5 Durum Wheet sheet 3 subplots file
+season_5 <- read.csv('/Users/kimberlyh/Desktop/season_metadata_upload/season5_wheat/input_data/Season5_DurumWheat_SubPlots.csv', stringsAsFactors = FALSE)

```

Comment:
be careful of absolute paths and code that will only work on your computer. This file isn't on my computer ... need to at least add a link to the source plus some text to show how to get this file from its canonical source on google docs to here. probably worthwhile to look into the googlesheets package that can read directly from the source (though last I tried it was a little tricky to supply credentials).